packages/teams-js/rollup.config.mjs

```diff
@@ -0,0 +1,58 @@
+// rollup.config.mjs
+
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+import terser from '@rollup/plugin-terser';
+import typescript from '@rollup/plugin-typescript';
+import nodePolyfills from 'rollup-plugin-polyfill-node';
+
+import version from './package.json' assert { type: 'json' };
+
+export default {
+  input: './src/index.ts',
+  output: {
+    dir: 'dist/esm',
```

Comment:
How is `esm` being used when it comes to publishing?

Reply:
Your above answer, answers this

Reply:
Can you please elaborate a bit more? I'm not sure I fully understand what you're asking

Reply:
Oh I was just curious how the `esm` bits were being pulled in and published and your answer about publishing both `esm` and `umd` when publishing to the NPM Feed answered that for me